pysat/instruments/icon_euv.py

```diff
@@ -208,13 +203,13 @@ def clean(inst, clean_level=None):
         L26_Flag = inst['ICON_L26_Flags']
```

Comment:
Suggest:
```
icon_flag = 'Flags' if 'Flags' in inst.variables.keys() else 'ICON_L26_Flags'
```
^ But I don't recall if `inst.variables.keys()` or `inst.variables` is the right call.

And I changed L26_Flags to icon_flag to enforce snake_case and make the variable more descriptive to new ICON users.

Reply:
should be inst.variables.  will update accordingly.  Thanks!